README.md

```diff
@@ -1,3 +1,29 @@
+[![Build](https://travis-ci.org/mozilla-services/autopush-rs.svg?branch=master)](https://travis-ci.org/mozilla-services/autopush-rs)
```

Comment:
I got some more badges if you want em https://raw.githubusercontent.com/mozilla-services/slog-mozlog-json/master/README.md

(we don't need no stinking badges?)